src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -883,41 +886,41 @@ private void ExitMode()
             }
 
             // Whether we've hit the end of input yet. If this returns true, ChCur will be zero.
-            private bool Eof => CurrentPos >= _charCount;
+            private bool Eof => _currentPosition >= _charCount;
 
             // The current position.
-            private int CurrentPos { get; set; }
+            private int _currentPosition;
```

Comment:
>_currentPosition [](http://example.com/codeflow?start=24&length=16)

Assuming this is because this is a major hot path; and field access is faster than property access? (I'd have hoped that the jitted would have inlined the property, but alas)

Reply:
very hot path for sure in the lexer
hard to tell if the improvement is major as the use of a field makes it disappear from the profiler